tests/paramgrill.c

```diff
@@ -108,17 +110,57 @@ static double g_compressibility = COMPRESSIBILITY_DEFAULT;
 static U32 g_blockSize = 0;
 static U32 g_rand = 1;
 static U32 g_singleRun = 0;
+static U32 g_optimizer = 0;
 static U32 g_target = 0;
 static U32 g_noSeed = 0;
-static ZSTD_compressionParameters g_params = { 0, 0, 0, 0, 0, 0, ZSTD_greedy };
+static ZSTD_compressionParameters g_params;
```

Comment:
explain it's going to be initialized somewhere else.
explain why it's guaranteed to not be used initialized.

note : the memory sanitizer and valgrind can typically be used to check that. Some static analyzer might also be able to prove it, thought success rate can vary.